packages/search/addon/components/navi-search-result/report.hbs

```diff
@@ -0,0 +1,12 @@
+<div {{did-insert this.search @query}}>
+    {{#if searchResult.content}}
+    <h1>{{this.title}}</h1>
+    <div class="search-result">
```

Comment:
Can we combine with `ul` below?

Reply:
What do we mean by combine?

Reply:
move the class to the `ul` and remove the `div`.